.semaphore/semaphore.yml

```diff
@@ -16,7 +16,8 @@ blocks:
         commands:
           - checkout
           - cache restore bundle-$SEMAPHORE_GIT_BRANCH-$SEMAPHORE_WORKFLOW_ID,bundle-$SEMAPHORE_GIT_BRANCH,bundle-master
-          - bundle install --path vendor/bundle
+          - bundle config set path 'vendor/bundle'
+          - bundle install
           - bundle clean --force
```

Comment:
Addresses [deprecation of --path](https://simplybusiness.semaphoreci.com/jobs/1f166def-87e9-4f4d-8dea-1f64ee7211b0#L87-L88)